### PR TITLE
Note about release notes in bug fixes

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**
FYI @gavin-norman-sociomantic

### CODE REVIEW

File: doc/library-maintainer.rst [hunk 1]
```diff
@@ -139,6 +139,9 @@ How to Make a Release
    * For minor and major releases, the full release notes text for the
      release, including descriptions of migration instructions, deprecations,
      new features, etc.
+   * If patch releases contains any release notes, those should be copied too.
```
Comment (gavin-norman-sociomantic) [inline doc/library-maintainer.rst]: I'd suggest melding this with the above:
"
For patch releases, links to the issues fixed in the release (or the PRs which fixed the issues), along with either a one-line description of what was fixed (e.g. the PR or issue title) or more detailed release notes providing a more detailed explanation about the importance of the fix and its potential consequences.
"

Comment (gavin-norman-sociomantic) [reply doc/library-maintainer.rst]: Actually, maybe this is better:
"
For patch releases, links to the issues fixed in the release (or the PRs which fixed the issues), along with the full release notes text including detailed explanations about the importance of the fixes and their potential consequences. (If a patch release does not have pre-written release notes, the PR or issue title may be included.)
"

**Comment by gavin-norman-sociomantic** [comment]
https://github.com/sociomantic-tsunami/neptune/blob/master/doc/announcements.md and https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst also need to be updated.

**Comment by mihails-strasuns-sociomantic** [comment]
I don't think announcements need to be updated, it already mentions abstract "description" for point releases.

**Comment by mihails-strasuns-sociomantic** [comment]
Updated with new wording and contributor instructions

**Comment by gavin-norman-sociomantic** [comment]
>I don't think announcements need to be updated, it already mentions abstract "description" for point releases.

I disagree. Rather than "A bullet list of issues fixed, including descriptions of the bug and what code was affected by it, links to the corresponding PRs or issues in github.", it should say "The collated release notes of the release.", the same as the other types of release.

**Comment by mihails-strasuns-sociomantic** [comment]
Updated

**Pull request merged**